e2e/test/scenarios/embedding/embedding-reproductions.cy.spec.js

```diff
@@ -494,6 +504,109 @@ describe("issues 20845, 25031", () => {
   });
 });
 
+describe.skip("27643", { tags: "@external" }, () => {
+  const PG_DB_ID = 2;
+
+  beforeEach(() => {
+    restore("postgres-12");
+    cy.signInAsAdmin();
+    resyncDatabase({
+      dbId: PG_DB_ID,
+    });
+    const TEMPLATE_TAG_NAME = "boolean_param";
+    withDatabase(PG_DB_ID, ({ INVOICES }) => {
+      /** @type {import("e2e/support/helpers").NativeQuestionDetails} */
```

Comment:
Same thing here, there could be conflicts if I convert this file to TypeScript now, and that isn't even easy, since there are a bunch of type errors, so it's kind of impossible at the moment.